README.md

```diff
@@ -143,6 +143,10 @@ Currently, this only supports a subset of PAMs where:
 * Supports reading version 1 and version 2
 * Supports writing version 2
 
+### Other formats

```

Comment:
This is not an image format but a color/pixel format and it is not at its right place in the README. We should list the pixel format supported by color/pixel_storage in a separate section

Reply:
So I just reverted the README, as I do not have a clear understanding of where to put it. But as Rgb565 is also not listed, that might be still ok :) 
If not, it would be great if you could give an example. I assume you would like to have something like a matrix then, but to be honest, I would need to do a lot more research then :)